index.html

```diff
@@ -5082,9 +5081,13 @@ <h2 id="AudioParam">
               <p>
                 Setting this attribute has the effect of assigning the
                 requested value to the <var>[[current value]]</var> slot, and
-                calling <code>setValueAtTime()</code> with the current
-                <code>AudioContext</code>'s <code>currentTime</code> and
-                <var>[[current value]]</var>.
+                calling the <a href=
+                "#dom-audioparam-setvalueattime">setValueAtTime()</a> method
```

Comment:
Nit: I think we should avoid this `href="#dom-audioparam..."` stuff. With contiguous idl, I think the right way is `<a data-link-for="AudioParam">setValueAtTime</a>()`.  (Not sure if the parens should be inside our outside that a tag.

Reply:
cool, I didn't know the right way.